core/templates/dev/head/css/oppia.css

```diff
@@ -2129,16 +2130,28 @@ div#ng-curtain {
   margin-left: -10px;
 }
 
-.oppia-logo {
+.oppia-logo-wide,
+.oppia-logo-small {
   display: inline-block;
   font-family: "Capriola", "Roboto", Arial, sans-serif;
   font-size: 21px;
   font-weight: 300;
   height: 40px;
-  margin-left: 6px;
   margin-top: 8px;
 }
-.oppia-navbar-profile-dropdown-toggle {
+.oppia-logo-wide {
+  margin-left: 25px;
+}
+.oppia-logo-small {
+  margin-left: 6px;
+}
+
+.nav > li > a.oppia-navbar-tabs {
```

Comment:
Should this be named oppia-navbar-tab instead, since it relates to a single tab?